addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -104,6 +104,9 @@
                                 <td name="td_quantity" class="text-end">
                                     <span t-field="line.product_uom_qty">3</span>
                                     <span t-field="line.product_uom">units</span>
+                                    <span t-if="line.product_packaging_id">
+                                        (<span t-field="line.product_packaging_qty" t-options='{"widget": "integer"}'/> <span t-field="line.product_packaging_id"/>)
+                                    </span>
```

Comment:
Why not the standard float widget since it's a float field ?
Unless the field only holds integer value, I don't understand why we would discard the decimal information in the report :thinking:

```suggestion
                                        (<span t-field="line.product_packaging_qty" t-options="{'widget': 'integer'}"/> <span t-field="line.product_packaging_id"/>)
                                    </span>
```
(nitpicking)

Reply:
It's one of those annoying "what the user wants to do" vs "what makes sense". Packagings aren't supposed to be sold as partial packagings (e.g. you can't buy 4 rolls of toilet paper by removing 6 from a packaging of 10 rolls and say you are buying 0.4 of a packaging of 10 unless it's a very strange supermarket), but due to multiple bugfixes demanding that we do, we don't strictly enforce it anymore. So we made drop the decimal info in the delivery slip because we expect non-integer values to be an exceptional use case.

Reply:
Ok, fine then, they will add the float support if they need it for their own instance :ok_hand: